configuration/config-txt/video.md

```diff
@@ -58,17 +21,24 @@ overscan_top=24
 overscan_bottom=24
 ```
 
+### hdmi_force_hotplug
+
+Setting `hdmi_force_hotplug` to `1` forces the HDMI output to be enabled, even if there is no display detected.
+
+### hdmi_ignore_hotplug
+
+Setting `hdmi_ignore_hotplug` to `1` prevents the HDMI output from being enabled, even if a display is attached.
 ### hdmi_ignore_edid
```

Comment:
Probably want a blank line above each heading, for clarity?

Reply:
Oops - fixed.